docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -0,0 +1,7 @@
+#Internal and external communication
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned in [[Polyglot systems with Lagom|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed.  When choosing the mode of communication for your microservices, keep in mind inter-service communication as well as external communication. 
```

Comment:
I like the distinction of three scenarios: intra-service, inter-service and external. It may be worth investigating the implications of each of the three and do a brief intro of each of them. Often, when discussing PubSub v Broker I'm missing that reference intro.

In any cases, not in this PR.